tests/trainer/logging_/test_train_loop_logging.py

```diff
@@ -31,7 +33,8 @@
 from tests.helpers.runif import RunIf
 
 
-def test__training_step__log(tmpdir):
+@mock.patch("pytorch_lightning.loggers.TensorBoardLogger.log_metrics")
+def test__training_step__log(mock_log_metrics, tmpdir):
```

Comment:
Can you write a new test instead? This test is already way too broad.

Reply:
done!